src/Microsoft.ML.Dnn/Microsoft.ML.Dnn.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="SciSharp.TensorFlow.Redist" Version="1.14.0" />
     <PackageReference Include="System.IO.FileSystem.AccessControl" Version="$(SystemIOFileSystemAccessControl)" />
     <PackageReference Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindows)" />
-    <PackageReference Include="TensorFlow.NET" Version="0.10.10" />
+    <PackageReference Include="TensorFlow.NET" Version="0.11.3" />
```

Comment:
Don't forget to also update the file in the `pkg`.  Usually we put all the dependency versions in a single file - https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props. That way they only need to be updated in a single place.